projects/examples/src/components/action-menu/action-menu.examples.module.ts

```diff
@@ -0,0 +1,29 @@
+/*!
+ * Copyright 2019 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { NgModule } from '@angular/core';
+
+import { ActionMenuComponent } from '@vcd/ui-components';
+import { Documentation } from '@vmw/ng-live-docs';
+import { ActionMenuExampleComponent } from './action-menu.example.component';
+import { ActionMenuExampleModule } from './action-menu.example.module';
+
+Documentation.registerDocumentationEntry({
+    component: ActionMenuComponent,
+    displayName: 'Action menu',
+    urlSegment: 'ActionMenu',
```

Comment:
action-menu

Please remember to be consistent, I've seen some kebabCasing but never TitleCase. Standard for URLs is `whatever-this-is`